src/pwncore/routes/ctf/__init__.py

```diff
@@ -11,6 +11,7 @@
 
 router = APIRouter(prefix="/ctf", tags=["ctf"])
 
+from . import start
```

Comment:
Why are you using dependency injection internally?
Is there no better way to do this?

Reply:
You can invert this relation with a non-prefixed router
https://fastapi.tiangolo.com/reference/apirouter/?h=api#fastapi.APIRouter--example